mne/viz/evoked.py

```diff
@@ -1828,7 +1828,7 @@ def plot_evoked_joint(
     %(picks_all)s
     exclude : None | list of str | 'bads'
```

Comment:
the code in #13157 shows that `None` isn't even valid input, so we should remove it here too:

```suggestion
    exclude : list of str | 'bads'
```